crates/cli/src/args.rs

```diff
@@ -95,6 +95,10 @@ pub struct Args {
     /// Arguments given to the Wasm module or the invoked function.
     #[clap(value_name = "ARGS")]
     func_args: Vec<String>,
+
+    /// Enable informational messages beyond warnings or errors.
+    #[clap(long = "verbose")]
+    verbose: bool,
```

Comment:
I am not 100% sure but I think it is important that the `func_args` field comes last since it takes a variable number of arguments. So please move the `verbose` field before it.

Reply:
Sounds fine. Change made.

Reply:
Also, if you'd prefer restricting the remaining fuel message to verbose mode already, let me know, and I can do that, too.